lib/clean.js

```diff
@@ -7,13 +7,12 @@ var defaults = {
   dest: 'dist'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
 
-  gulp.task('clean', function (done) {
-    del(dest).then(function () {
-      done();
-    });
+  gulp.task('clean', done => {
+    del(dest)
+      .then(() => done());
```

Comment:
@lyzadanger: I actually did try and it didn't seem to be working. Tests would pass, but when I actually tried to use the `clean` task, it would only half complete (delete the `deleteMe/` directory properly) then error out. 

Output is here:

```
$ gulp clean
[14:11:53] Using gulpfile ~/Development/projects/cloudfour/test-2/gulpfile.js
[14:11:53] Starting 'clean'...
[14:11:53] 'clean' errored after 7.22 ms
[14:11:53] Error: /Users/Rodriguez/Development/projects/cloudfour/test-2/deleteMe
    at formatError (/Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/gulp/bin/gulp.js:195:15)
    at emitOne (events.js:77:13)
    at Gulp.emit (events.js:169:7)
    at Gulp.Orchestrator._emitTaskDone (/Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/orchestrator/index.js:264:8)
    at /Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/orchestrator/index.js:275:23
    at finish (/Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/Users/Rodriguez/Development/projects/cloudfour/test-2/node_modules/orchestrator/lib/runTask.js:29:3)
```